templates/zerver/integrations/jenkins.md

```diff
@@ -27,32 +29,47 @@ Jenkins.)
 ### Configure a post-build action for your project
 
 Once you've done that, it's time to configure one of your
-projects to use the Zulip notification plugin. From your
+freestyle projects to use the Zulip notification plugin. From your
 project page, click **Configure** on the left sidebar. Scroll to
 the bottom until you find the section labeled **Post-build
 Actions**. Click the dropdown and select **Zulip Notification**.
-It should look as below. Then click **Save**.
+It should look as below. If you don't want to use defaults from
+global configuration you can set destination stream and topic
+(if you don't set the topic here or in global configuration, project
+name will be used as topic instead).
+Then click **Save**.
 
 ![](/static/images/integrations/jenkins/003.png)
 
 When your builds fail or succeed, you'll see a commit message
-with a topic that matches the project name (in this case
-"SecretProject").
+with a topic you configured or one that matches the project name
+(in this case "SecretProject"). If you configured topic to post to,
+the message will also include project name.
 
 {!congrats.md!}
 
 ![](/static/images/integrations/jenkins/004.png)
 
+### Advanced use cases
+
+Besides the **Zulip Notification** post-build action, the plugin
+also supports **Zulip Send** step you can use anywhere throughout your build
+and Jenkins pipeline jobs.
+You can read more in the [plugin's readme](https://github.com/zulip/zulip-jenkins-plugin).
+
 ### Troubleshooting
 
 1. Did you set up a post-build action for your project?
 
-2. Does the stream you picked (e.g. `jenkins`) already exist?
+1. Does the stream you picked (e.g. `jenkins`) already exist?
    If not, add yourself to it and try again.
```

Comment:
I'd probably change the last sentence to: `If not, create the stream and make sure you're subscribed to it.`